houdini/python2.7libs/hda/software.py

```diff
@@ -100,9 +99,9 @@ def _plugins_for_host_package(package):
 
     """
     result = {}
-    for tool in [p for p in package.keys() if not p == "package"]:
-        versions = package[tool].keys()
-        for version in versions:
+    for tool in [p for p in package if not p == "package"]:
+        # versions = package[tool]
```

Comment:
Can you remove this commented out line?

Reply:
yeah I caught that too after the PR. Cheers